cohort-engine/src/main/java/com/ibm/cohort/engine/EvaluationResultCallback.java

```diff
@@ -1,15 +1,6 @@
-/*
```

Comment:
Why remove copyright?

Reply:
Good point and not the only one missing. I'll do a pass to add copyrights (again). Not sure why my Eclipse templating isn't working.